base/inc/fields/js/media-field.js

```diff
@@ -1,8 +1,9 @@
 ( function( $ ) {
 
     $(document).on( 'sowsetupformfield', '.siteorigin-widget-field-type-media', function(e) {
-        var $media = $(this).find('> .media-field-wrapper');
-        var $field = $media.closest('.siteorigin-widget-field');
+		var $field = $( this );
+        var $media = $field.find('> .media-field-wrapper');
```

Comment:
Mmm, looks like there's a mix of space and tab indents in a few of these field JS files. So maybe we should just leave this for now, and reformat all of them in a separate commit after the merge.